skins/oasis/css/core/page.scss

```diff
@@ -281,6 +279,60 @@
 	}
 }
 
+#WikiaRecentActivity {
```

Comment:
.module overrides my styles when using class selector, we'll clean it after we restyle the rest of rail modules